.github/workflows/continuous-deploy.yml

```diff
@@ -231,6 +233,11 @@ jobs:
               echo "Tests passed with teardown error"
               exit 0
           else
+              if [ "$EXIT_CODE" -ne 0 ]; then
+                echo "test_success=false" >> $GITHUB_OUTPUT
+              else
+                echo "test_success=true" >> $GITHUB_OUTPUT
+              fi
               exit $EXIT_CODE
```

Comment:
This `exit $EXIT_CODE` would exit with a non-zero exit code, marking the github step as having failed, which would cancel subsequent jobs from running.

Is the `outputs.test_success` necessary? Wouldn't the job failing handle it?